integrations-and-supported-tools/pytorch/scripts/Neptune_Pytorch_Support.py

```diff
@@ -0,0 +1,115 @@
+from numpy.random import permutation
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+from torchvision import datasets, models, transforms
+import neptune.new as neptune
+import numpy as np
+
+
+def main():
```

Comment:
I don't think we need to add main in those examples. 
It adds unnecessary complexity when reading it. 
We'll never use it as a module so I don't think we should overengineer it.

Reply:
Forgot to switch it back. I will fix that.
Although, I did that becuase the script was not running on my machine without it.
It kept giving me an error.

Reply:
Forgot to switch it back. I will fix that.
Although, I did that becuase the script was not running on my machine without it.
It kept giving me an error.